client/dashboard/domains/domain-overview/actions.tsx

```diff
@@ -44,6 +61,21 @@ export default function Actions() {
 		[ disconnectDomain, createSuccessNotice, createErrorNotice ]
 	);
 
+	const onDeleteConfirm = useCallback(
+		() =>
+			purchase &&
+			deleteDomain( purchase.ID, {
+				onSuccess: () => {
+					createSuccessNotice( __( 'The domain deletion has been completed.' ), {
```

Comment:
I think in our original snackbars we designed with Matt W these messages were a bit more parameterised would you say @fushar?

```suggestion
					createSuccessNotice( __( '%s was successfully deleted.' ), {
```

Not a blocker, just a general comment about the snackbars.

Reply:
Yeah, I think so, confirmation snackbar (especially for destructive actions) should ideally explicitly mention the object names.

Reply:
Thanks for the feedback.

Since we have similar usage across all domains, I've created a separate ticket with low priority to review the usage within the domains project.

DOTDASH-357/notice-snackbar-review-usage-within-domainsdomain-product-part